docker-compose.yml

```diff
@@ -0,0 +1,91 @@
+---
+version: '3.7'
+services:
+
+  readeef:
+    image: readeef:alpine3.10-go1.13
```

Comment:
looks like this is currently called urandom/readeef

Reply:
I will fix it, did you create an account on hub.docker.com ? or shall I put mine

Reply:
I think I managed to create an org/repo there just now. i think i gave you full permission there

Reply:
I updated the namespace to unrandom just now.

I have a question about the ui, it seems that we have to copy some files to final container ?
https://github.com/x0rzkov/readeef/blob/dockerize/Dockerfile#L30
https://github.com/x0rzkov/readeef/blob/dockerize/.docker/readeef/config/readeef.toml#L5-L6

Reply:
Do you have a full config file example ? with all attributes.

Reply:
> I updated the namespace to unrandom just now.
> 
> I have a question about the ui, it seems that we have to copy some files to final container ?
> https://github.com/x0rzkov/readeef/blob/dockerize/Dockerfile#L30
> https://github.com/x0rzkov/readeef/blob/dockerize/.docker/readeef/config/readeef.toml#L5-L6

we shouldn't need to copy the ui. it's already embedded in the readeef binary.

Reply:
weird... I have the following login page https://imgur.com/a/IxdeSZP
What are the default credentials ?